x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -3974,14 +3974,14 @@
     "xpack.actions.serverSideErrors.unavailableLicenseErrorMessage": "操作类型 {actionTypeId} 已禁用，因为许可证信息当前不可用。",
     "xpack.actions.serverSideErrors.unavailableLicenseInformationErrorMessage": "操作不可用 - 许可信息当前不可用。",
     "xpack.actions.urlWhitelistConfigurationError": "目标 {field}“{value}”在 Kibana 配置 xpack.actions.whitelistedHosts 中未列入白名单",
-    "xpack.advancedUiActions.components.actionWizard.changeButton": "更改",
-    "xpack.advancedUiActions.customizePanelTimeRange.modal.addToPanelButtonTitle": "添加到面板",
-    "xpack.advancedUiActions.customizePanelTimeRange.modal.cancelButtonTitle": "取消",
-    "xpack.advancedUiActions.customizePanelTimeRange.modal.optionsMenuForm.panelTitleFormRowLabel": "时间范围",
-    "xpack.advancedUiActions.customizePanelTimeRange.modal.removeButtonTitle": "删除",
-    "xpack.advancedUiActions.customizePanelTimeRange.modal.updatePanelTimeRangeButtonTitle": "更新",
-    "xpack.advancedUiActions.customizeTimeRange.modal.headerTitle": "定制面板时间范围",
-    "xpack.advancedUiActions.customizeTimeRangeMenuItem.displayName": "定制时间范围",
+    "xpack.uiActionsEnhanced.components.actionWizard.changeButton": "更改",
```

Comment:
But should sorting of this change? 🤔

Reply:
I never sorted translations before, and it worked. But maybe the correct way is to sort them, maybe not 🤷‍♂️